test/TestProjects/MgmtMockAndSample/tests/Generated/Samples/Sample_DiskEncryptionSetCollection_CreateOrUpdate_CreateADiskEncryptionSet.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Models;
+using Azure.ResourceManager.Resources;
+using MgmtMockAndSample.Models;
+
+namespace MgmtMockAndSample
+{
+    public partial class Sample_DiskEncryptionSetCollection_CreateOrUpdate_CreateADiskEncryptionSet
+    {
+        // Create a disk encryption set.
+        [NUnit.Framework.Test]
+        [NUnit.Framework.Ignore("Only verifying that the sample builds")]
+        public async Task CreateOrUpdate()
+        {
+            // this example is just showing the usage of "DiskEncryptionSets_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
+
+            // authenticate your client
+            ArmClient client = new ArmClient(new DefaultAzureCredential());
+
+            // this example assumes you already have this ResourceGroupResource created on azure
+            // for more information of creating ResourceGroupResource, please refer to the document of ResourceGroupResource
+            string subscriptionId = "{subscription-id}";
+            string resourceGroupName = "myResourceGroup";
+            ResourceIdentifier resourceGroupResourceId = ResourceGroupResource.CreateResourceIdentifier(subscriptionId, resourceGroupName);
+            ResourceGroupResource resourceGroupResource = client.GetResourceGroupResource(resourceGroupResourceId);
+
+            // get the collection of this DiskEncryptionSetResource
+            MgmtMockAndSample.DiskEncryptionSetCollection collection = resourceGroupResource.GetDiskEncryptionSets();
+
+            // invoke the operation
+            string diskEncryptionSetName = "myDiskEncryptionSet";
+            MgmtMockAndSample.DiskEncryptionSetData data = new DiskEncryptionSetData()
+            {
+                Identity = new ManagedServiceIdentity("SystemAssigned"),
+                EncryptionType = DiskEncryptionSetType.EncryptionAtRestWithCustomerKey,
+                ActiveKey = new KeyForDiskEncryptionSet(new Uri("https://myvmvault.vault-int.azure-int.net/keys/{key}"))
+                {
+                    SourceVaultId = new ResourceIdentifier("/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.KeyVault/vaults/myVMVault"),
+                },
+            };
+            ArmOperation<MgmtMockAndSample.DiskEncryptionSetResource> lro = await collection.CreateOrUpdateAsync(WaitUntil.Completed, diskEncryptionSetName, data);
+            MgmtMockAndSample.DiskEncryptionSetResource result = lro.Value;
```

Comment:
Add some comments to describe the resource here - you could invoke other method on this instance.